components/Dashboard/Cards/SpaceCardModal.js

```diff
@@ -86,15 +88,18 @@ const SpaceCardModal = ({ handleClose, open, children, friends, participants, ho
   const router = useRouter();
   const [client, setClient] = useRecoilState(clientState.client);
   const [roomIsLoading, setRoomIsLoading] = useState(false);
-
-  // const gqlClient = useApolloClient();
-  // const { loading, error, data } = useQuery(GET_SESSION_USER);
-  // console.debug('data', data);
-
+  const [addUserToSpace] = useMutation(ADD_USER_TO_SPACE);
+  console.debug('participants in space modal', participants);
```

Comment:
Can you remove these console debugs from production? We usually only keep them in prod if they're in a function call (usually one that makes and api call). Otherwise the console debug will execute in every rerender, cluttering up the console.